src/azul/plugins/metadata/anvil/indexer/transform.py

```diff
@@ -464,7 +474,37 @@ def _complete_dataset_keys(cls) -> AbstractSet[str]:
         return cls.field_types()['datasets'].keys()
 
 
-class ActivityTransformer(BaseTransformer):
+class PartitionedTransformer(BaseTransformer, metaclass=ABCMeta):
```

Comment:
I don't understand this class name. BaseTransformer handles the the partitioning, and all subclasses must play well with that. IOW, SingletonTransformer is also "partitioned" aka supports partitioning. This class hierarchy seems to suggest that it is not.

Reply:
The name was chosen for consistency with [the precedent established by the HCA plugin](https://github.com/DataBiosphere/azul/blob/225b31d8c2f3e7996680c59e24cfd1b7f3944ade/src/azul/plugins/metadata/hca/indexer/transform.py#L1363). I believe your criticism is equally applicable to that module.

Reply:
Decided at PL this class can be removed, with its implementation of `_list_entities` moved to `BaseTransformer`